Kernel/EFIPrekernel/Arch/aarch64/Boot.cpp

```diff
@@ -6,25 +6,262 @@
 
 #include <AK/Types.h>
 
+#include <Kernel/Arch/aarch64/ASM_wrapper.h>
+#include <Kernel/Arch/aarch64/Registers.h>
+#include <Kernel/Sections.h>
+
 #include <Kernel/EFIPrekernel/Arch/Boot.h>
+#include <Kernel/EFIPrekernel/Arch/MMU.h>
+#include <Kernel/EFIPrekernel/Panic.h>
 #include <Kernel/EFIPrekernel/Runtime.h>
+#include <Kernel/EFIPrekernel/VirtualMemoryLayout.h>
 
 namespace Kernel {
 
+// This function has to fit into one page as it will be identity mapped.
```

Comment:
Can we assert the size limit somehow?
Maybe with a trailing symbol, but not sure how to make it a static assert from there

Reply:
Only with an external script. I don't think that's worth the extra effort.